packages/typesafe-db/src/customers.ts

```diff
@@ -122,63 +119,98 @@ function getCustomerList(
+    .leftJoin('editedCustomerData as cstED', 'cstED.customerId', 'cst.id')
     .leftJoinLateral(joinTxsTotals, join => join.onTrue())
     .leftJoinLateral(joinLatestTx, join => join.onTrue())
-    .select(({ eb, fn, val, ref }) => [
-      'c.id',
-      'c.phone',
-      'c.authorizedOverride',
-      'c.frontCameraPath',
-      'c.frontCameraOverride',
-      'c.idCardPhotoPath',
-      'c.idCardPhotoOverride',
-      selectNewestIdCardData(eb, ref).as('idCardData'),
-      'c.idCardDataOverride',
-      'c.email',
-      'c.usSsn',
-      'c.usSsnOverride',
-      'c.sanctions',
-      'c.sanctionsOverride',
+    .select(({ eb, fn, val }) => [
+      'cst.id',
+      'cst.phone',
+      'cst.authorizedOverride',
+      'cst.frontCameraPath',
+      'cst.frontCameraOverride',
+      'cst.idCardPhotoPath',
+      'cst.idCardPhotoOverride',
+      selectNewestIdCardData(eb).as('idCardData'),
+      'cst.idCardDataOverride',
+      'cst.email',
+      'cst.usSsn',
+      'cst.usSsnOverride',
+      'cst.sanctions',
+      'cst.sanctionsOverride',
       'txStats.totalSpent',
       'txStats.totalTxs',
-      ref('lastTx.fiatCode').as('lastTxFiatCode'),
-      ref('lastTx.fiat').as('lastTxFiat'),
-      ref('lastTx.txClass').as('lastTxClass'),
+      'lastTx.fiatCode as lastTxFiatCode',
+      'lastTx.fiat as lastTxFiat',
+      'lastTx.txClass as lastTxClass',
       fn<Date>('GREATEST', [
-        'c.created',
+        'cst.created',
         'lastTx.created',
-        'c.phoneAt',
-        'c.emailAt',
-        'c.idCardDataAt',
-        'c.frontCameraAt',
-        'c.idCardPhotoAt',
-        'c.usSsnAt',
-        'c.lastAuthAttempt',
+        'cst.phoneAt',
+        'cst.emailAt',
+        'cst.idCardDataAt',
+        'cst.frontCameraAt',
+        'cst.idCardPhotoAt',
+        'cst.usSsnAt',
+        'cst.lastAuthAttempt',
       ]).as('lastActive'),
-      eb('c.suspendedUntil', '>', fn<Date>('NOW', [])).as('isSuspended'),
+      eb('cst.suspendedUntil', '>', fn<Date>('NOW', [])).as('isSuspended'),
       fn<number>('GREATEST', [
         val(0),
         fn<number>('date_part', [
           val('day'),
-          eb('c.suspendedUntil', '-', fn<Date>('NOW', [])),
+          eb('cst.suspendedUntil', '-', fn<Date>('NOW', [])),
         ]),
       ]).as('daysSuspended'),
     ])
-    .where('c.id', '!=', ANON_ID)
+    .where('cst.id', '!=', ANON_ID)
     .$if(options.withCustomInfoRequest, qb =>
       qb.select(({ eb, ref }) =>
         jsonArrayFrom(
           eb
             .selectFrom('customersCustomInfoRequests')
             .selectAll()
-            .where('customerId', '=', ref('c.id')),
+            .where('customerId', '=', ref('cst.id')),
         ).as('customInfoRequestData'),
       ),
     )
     .orderBy('lastActive', 'desc')
     .execute()
 }
 
-export { getCustomerList }
+function searchCustomers(searchTerm: string, limit: number = 20): Promise<any> {
+  const searchPattern = `%${searchTerm}%`
```

Comment:
Would be better if this used [Kysely's `sql`](https://kysely-org.github.io/kysely-apidoc/interfaces/Sql.html):

```ts
sql`%${searchTerm}%`
```